pkg/didcomm/protocol/introduce/service.go

```diff
@@ -22,3 +25,28 @@ const (
 	// AckMsgType defines the introduce ack message type.
 	AckMsgType = IntroduceSpec + "ack"
 )
+
+// Service for introduce protocol
+type Service struct {
+	service.Action
```

Comment:
Based on the [comment](https://github.com/hyperledger/aries-framework-go/pull/449#discussion_r333091231), I see MsgEvent related APIs are removed. But looks like it's missing from here. ie. `service.Message`.

Reply:
Done! thanks) good catch